src/message/fetchActions.js

```diff
@@ -314,17 +294,11 @@ export const doInitialFetch = () => async (dispatch: Dispatch, getState: GetStat
   }
 
   dispatch(realmInit(initData, new ZulipVersion(serverSettings.zulip_version)));
-  dispatch(fetchTopMostNarrow());
   dispatch(initialFetchComplete());
   dispatch(startEventPolling(initData.queue_id, initData.last_event_id));
 
   dispatch(fetchPrivateMessages());
 
-  const session = getSession(getState());
-  if (session.lastNarrow) {
-    dispatch(fetchMessagesInNarrow(session.lastNarrow));
-  }
-
```

Comment:
Huh, I hadn't quite spotted before that we do the "topmost narrow" fetch *before* INITIAL_FETCH_COMPLETE and the start of polling, and the "last narrow" fetch *after* those. (And the PM fetch after them as well.)

Seems likely there's a subtle bug in there, one way or the other.